src/custom-elements.js

```diff
@@ -451,6 +459,9 @@ let Deferred;
         const walker = createTreeWalker(root);
         do {
           const node = /** @type {!HTMLElement} */ (walker.currentNode);
```

Comment:
FWIW I think this annotation is incorrect; you may have non-HTMLElements in a document, usually SVG.